src/tests/e2e/receipt.test.e2e.js

```diff
@@ -0,0 +1,569 @@
+          ],
+          [
+            // should not have any errors
+            {
+              action: {
+                type: 'ENQUEUE_SNACKBAR',
+                notification: {
+                  options: {
+                    variant: 'error'
+                  }
+                }
+              }
+            }
+          ]
+        ),
+        page.waitForNavigation({
+          waitUntil: 'networkidle0'
+        }),
+        loginPage.login(
+          process.env.E2E_TEST_GOOGLE_LOGIN_USERNAME,
+          process.env.E2E_TEST_GOOGLE_LOGIN_PASSWORD,
+          true
+        )
+      ])
+      const receiptPage = new ReceiptPage()
+
+      const title = await receiptPage.getReceiptFormInfo('title')
+      expect(title).toEqual('Transfer Sent')
+
+      const sender = await receiptPage.getReceiptFormInfo('sender')
+      expect(sender.name).toEqual(transfer.senderName)
+      expect(sender.email).toEqual(transfer.sender)
+
+      const recipient = await receiptPage.getReceiptFormInfo('recipient')
+      expect(recipient.name).toEqual(transfer.receiverName)
+      expect(recipient.email).toEqual(transfer.destination)
+
+      const senderAccount = await receiptPage.getReceiptFormInfo('senderAccount')
+      const receiptSenderAccount = JSON.parse(transfer.senderAccount)
+      expect(senderAccount.walletType.toLowerCase()).toEqual(receiptSenderAccount.walletType)
+      expect(senderAccount.platformType.toLowerCase()).toEqual(receiptSenderAccount.platformType)
+      expect(senderAccount.address).toEqual(receiptSenderAccount.address)
+
+      const transferAmount = await receiptPage.getReceiptFormInfo('transferAmount')
+      expect(transferAmount.transferAmount).toEqual(transfer.transferAmount)
+      expect(transferAmount.currencyAmount).toEqual(transfer.transferFiatAmountSpot)
+      expect(transferAmount.symbol).toEqual(getCryptoSymbol(transfer.cryptoType))
+
+      const { securityAnswer } = await receiptPage.getReceiptFormInfo('securityAnswer')
+      expect(securityAnswer).toEqual(Base64.decode(driveTransfer.password))
+
+      const { message } = await receiptPage.getReceiptFormInfo('sendMessage')
+      expect(message).toEqual(transfer.sendMessage)
+
+      const { sendOn } = await receiptPage.getReceiptFormInfo('sendOn')
+      const expectTime = moment.unix(driveTransfer.sendTimestamp).format('MMM Do YYYY, HH:mm:ss')
+      expect(sendOn).toEqual(`Sent on ${expectTime}`)
+
+      const explorerPage = await getNewPopupPage(browser, async () => {
+        await receiptPage.dispatchActions('openSendExplorer')
+      })
+      expect(explorerPage.url()).toEqual(`https://rinkeby.etherscan.io/tx/${transfer.sendTxHash}`)
+      await explorerPage.close()
+
+      const { transferId } = await receiptPage.getReceiptFormInfo('transferId')
+      expect(transferId).toEqual(transfer.transferId)
+    },
+    timeout
+  )
+
+  it(
+    `Transfer ID: ${transferDataList[2].transferId} (SEND_CONFIRMED_RECEIVE_CONFIRMED) SENDER`,
```

Comment:
is it SEND_CONFIRMED_RECEIVE_PENDING?

Reply:
It is SEND_CONFIRMED_RECEIVE_CONFIRMED, "transfer completed"